### DISCUSSION TITLE
Raspberry pi4 8gb

### DISCUSSION THREAD

**Discussion by lucaxxaa** [indomus/forum#434] [Hardware & Firmware]
Buongiorno attualmente uso ora raspberry pi4 da 4 giga , volevo prendere 8 giga in quanto il mio vecchio usarlo come console.

Il Raspberry da 8 giga a 64 bit, funziona bene con home assistant e la propria versione a 64bit?

Posso fare e utilizzare il backup del mio attuale home assistant a 32 bit e portarlo sul nuovo a 64 bit?

Grazie

**Comment by indomus** [comment]
Se ti riferisci [a questo](https://amzn.to/3q9YUUe) (et similia), va più che bene, funziona perfettamente sia a 32 che 64. Sì, puoi certamente ripristinare il backup come da te indicato, non c'è alcun problema, tranquillo :)

**Reply by lucaxxaa** [reply]
Ti ringrazio allora visto che comunque devo ordinare un Raspberry Pi 4, ordine 8 GB ed userò quello per home assistant.
Ma se uso home assistant 32 bit, sfrutta comunque gli 8 GB di RAM.
E se installo home-assistant a 64 bit, posso ripristinate per il mio bakup attuale che è una 32 bit?

**Comment by maxcanna** [comment]
Io fossi in te risparmierei i soldi 🤑
